### ISSUE TITLE
Iter_messages() halts on some messages

### ISSUE THREAD

**Issue by iamfamily562** [LonamiWebs/Telethon#1281]
Hi,
I'm trying to get messages from a channel; but it halts on some messages.
For example, when i get messages from channel:"newchanlkomakmaeeshat" it gets messages until it reaches to message with id 34395, then it halts.
Example code:
```
input_channel = client.get_entity('newchanlkomakmaeeshat')
msgs = client.iter_messages(input_channel, offset_id=34395, limit=2)
for message in msgs:
    print(message.id)
```
Could you please help me?

**Comment by Lonami** [comment]
The code you provided does not get stuck for me. It may be a `FloodWaitError`. `INFO` logging would point this out. Please enable `logging`.

**Comment by iamfamily562** [comment]
Thank you for your reply.
I enabled logging, result exists in attachment.

[log.txt](https://github.com/LonamiWebs/Telethon/files/3652700/log.txt)

I must add that program does not stop and continue until user stops it.

**Comment by Lonami** [comment]
You are using an old version. This was (most likely) fixed by #1241.

**Comment by Lonami** [comment]
For what is worth, the faulty date is the following:
```python
>>> print(client.get_messages('newchanlkomakmaeeshat', ids=34393)
...       .web_preview.cached_page.blocks[2].published_date)
...
1952-08-28 21:40:30+00:00
```

The solution as I said is upgrading the library.
